src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/BookingExtractionService.kt

```diff
@@ -108,7 +111,15 @@ class BookingExtractionService(
       dates[LTD] = sentenceCalculation.latestTransferDate!!
     }
 
-    if (!sentenceIsOrExclusivelyBotus) {
+    if (sentence.isOrExclusivelyBotus()) {
+      if (sentence is BotusSentence && sentence.latestTusedDate != null && sentenceCalculation.expiryDate.isBefore(
+          sentence.latestTusedDate,
+        )
+      ) {
+        dates[TUSED] = sentence.latestTusedDate!!
+        historicalTusedSource = sentence.latestTusedSource!!
```

Comment:
I think that can cause some problems for the manual entry journey

Reply:
This is what @Matt changed, I suspect it's here due to merge conflct

Reply:
I only changed the function name from isBotus to isOrExclusivelyBotus 
isBotus was a little misleading, as it consecutive sentences were "isBotus" if all sentence parts were BOTUS sentences.

Reply:
Yea this was from a merge conflict, although it has no impact on the calculation because the logic had just been moved below.

It won't affect the manual calc journey as the ESED calculation doesn't come through the booking extraction service